src/data/types.ts

```diff
@@ -38,7 +38,7 @@ export type PhotoId = string | undefined
 export type PreviousOrNext = null | PhotoListItem
 
 export interface PhotoData {
-  id: PhotoId
+  id?: PhotoId
```

Comment:
This seems wrong, photos should always have an `id`.

Reply:
You are correct, I don't recalled the reason for me to make it conditional :(, I changed it without problems :)